doc/visual-programming/source/widgets/visualize/boxplot.rst

```diff
@@ -33,15 +33,12 @@ outliers, and alike.
 
 For continuous attributes the widget displays:
 
-4. The mean (the dark blue vertical line)
+4. The mean (the dark blue vertical line). The thin blue line represents the `standard deviation of the mean <https://en.wikipedia.org/wiki/Standard_deviation#Standard_deviation_of_the_mean>`_.
```

Comment:
"Standard deviation of the mean" describes the precision of the computed mean, which is not the case here. With standard deviation we report the dispersion of values around the computed mean. (http://mathworld.wolfram.com/StandardDeviation.html)